MAINTAINERS.yml

```diff
@@ -1876,6 +1868,8 @@ Release Notes:
     - teburd
   files:
     - drivers/timer/
+    - tests/kernel/timer/
+    - tests/drivers/timer/
```

Comment:
This looks unrelated

Reply:
it is, but does this now justify another PR to this non-code-changing PR?

Reply:
another PR no, but another commit would be nice